cogs/ahk/ahk.py

```diff
@@ -743,7 +743,34 @@ async def solved(self, inter: disnake.AppCmdInter):
 
         await inter.channel.edit(
             archived=True,
-            applied_tags=inter.channel.applied_tags + [solved_tag],
+            applied_tags=inter.channel.applied_tags + ([] if solved_tag in inter.channel.applied_tags else [solved_tag]),
+        )
+        
+    @commands.slash_command(description="Unark your post as solved.", guild_ids=[AHK_GUILD_ID])
+    async def unsolved(self, inter: disnake.AppCmdInter):
```

Comment:
all of the checks here are in common with /solved, at this point it's better suited as a commands check instead of being duplicated